### ISSUE TITLE
Video and voice calls do not go through wired headphones 

### ISSUE THREAD

**Issue by codejeet** [signalapp/Signal-Android#12622]
- [ x] I have searched open and closed issues for duplicates
- [x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Video and voice calls do not send audio through connected wired headphones. Instead, audio goes through the earpiece speaker

### Steps to reproduce
- connect wired headphone
- join video or voice call

**Actual result:** 
Audio only comes out from earpiece speaker 
**Expected result:** 
Audio comes from headphones 

### Device info

**Device:** Pixel 4A 5G
**Android version:** 13
**Signal version:** 6.1.4

### Link to debug log

https://debuglogs.org/android/6.1.4/f7562a4c92d692ab925feb0badb964d556e9a3b902208e75a98183b3fc36617b

**Comment by nicholas-signal** [comment]
Hi there, I appreciate the debug log. Quick question: did your headphones have a mic?

**Comment by MauRamirez1211** [comment]
> Hi there, I appreciate the debug log. Quick question: did your headphones have a mic?

I have the exact same issue. The 2 headphones I tried have no microphone. When connected I can hear the voice messages but as soon as a video or voice call is started it goes directly to the front earpiece speaker. It seems to be related to an update on the media output that started with Android 12 and above. I am currently enrolled as a participant for the beta versions in Signal in hope that a beta version with the fix were to be released.

**Device Info**
Device: Samsung A03s
Android Version: 12
Signal Version: 6.3.2

**Comment by kc2853** [comment]
Device info: samsung SM-A528B
Android version: 12
Signal version: 6.2.3

Same situation. Can hear voice messages no problem. Sound goes to earpiece speaker when joining calls. Speaker mode works totally fine. I definitely remember things working properly before with the same wired headphones. Much appreciated.

**Comment by junoslukan** [comment]
I can confirm the bug on the following device with wired headphones with no microphone. I have the same experience: I can hear voice messages, but video and voice calls relay audio from the earpiece speaker.

I would add, however, that I only noticed the bug about a month ago (but don't often call with wired headphones). I did not experience this previously, even though I had used Android 12 for about a year. So, I don't think that it is related to any changes introduced with Android 12.

Device: Pixel 4A (4G)
Android version: 13
Signal version: 6.2.4
